### PR TITLE
Add accomodation model and orchid admin for accomodation crud

### LINKED ISSUE #14: Create `Accommodation` model
The lodging rooms available at the center will be represented by an entity containing essential details. This entity will include information about the `name`, `location`, `description`, and `availability` of each room. Additional properties may also be included as needed.

Other complementary entities such as `AccommodationImage` and `Feature/Amenity` will be created to establish relationships with the `Accomodation` entity.

### PR DESCRIPTION
**Pull request by ExcelDurant**
closes #14

### CODE REVIEW

File: database/migrations/2023_06_24_093715_create_accomodations_table.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        Schema::create('accomodations', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('country');
+            $table->string('city');
+            $table->string('quarter');
+            $table->longText('description');
+            $table->boolean('available')->default(true);
+            $table->timestamps();
```
Comment (Fenn-CS) [inline database/migrations/2023_06_24_093715_create_accomodations_table.php:22]: All of these are supposed to be rooms in the same building but I think you can allow these fields.

Add `accommodation_number` and make everything nullable except for `name` and `accommodation_number`

**Review by Fenn-CS — CHANGES_REQUESTED** [review]
Looks good for a start.

`Accommodation` should have `photos` so this means you have to:

- Create `Photo` model
- Create a polymorphic one-to-many relationship between `Accommodation` and `Photo`

Take note that the photo model can eventually be used with other models as well.

**Comment by Fenn-CS** [comment]
I don't know if `Orchid` has an attachment model that can serve the same purpose as photos, if so we can just use it but they should be a place for the user to control that on the dashboard.

**Comment by ExcelDurant** [comment]
> I don't know if `Orchid` has an attachment model that can serve the same purpose as photos, if so we can just use it but they should be a place for the user to control that on the dashboard.

`Orchid` uses an attachment model to handle picture uploads

File: app/Orchid/Screens/Accomodation/AccomodationEditScreen.php [hunk 2]
```diff
@@ -128,9 +128,14 @@ public function layout(): iterable
                         ->title('no'),
                 ]),
 
+                Input::make('accomodation.accommodation_number')
```
Comment (Fenn-CS) [inline app/Orchid/Screens/Accomodation/AccomodationEditScreen.php:131]: Can this take multiple images? or just one?

Comment (ExcelDurant) [reply app/Orchid/Screens/Accomodation/AccomodationEditScreen.php:131]: multiple images can be uploaded for `Accomodation`

**Review by Fenn-CS — APPROVED** [review]

**Pull request merged**